File: src/Iniciante/1038.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <math.h>

int main()
{
    setlocale(LC_ALL, "pt-br");
    int codigo, quantidade;
    float total;
    scanf("%d %d", &codigo, &quantidade);

    if (codigo == 1)
    {
        total = quantidade * 4;
        printf("Total: R$ %.2f\n", total);
    }

    if (codigo == 2)
    {
        total = quantidade * 4.5;
        printf("Total: R$ %.2f\n", total);
    }

    if (codigo == 3)
    {
        total = quantidade * 5;
        printf("Total: R$ %.2f\n", total);
    }

    if (codigo == 4)
    {
        total = quantidade * 2;
        printf("Total: R$ %.2f\n", total);
    }

    if (codigo == 5)
    {
        total = quantidade * 1.5;
        printf("Total: R$ %.2f\n", total);
    }

    return 0;
}

File: src/Iniciante/1038.java
package Iniciante;

import java.util.Scanner;



public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double quantidade;
        int codigo;
        Double total;
        codigo = input.nextInt();
        quantidade = input.nextDouble();

        switch (codigo) {
            case 1:
                total = quantidade * 4;
                String t1 = String.format("%.2f", total);
                System.out.println("Total: R$ " + t1);
                break;
            case 2:
                total = quantidade * 4.5;
                String t2 = String.format("%.2f", total);
                System.out.println("Total : R$ " + t2);
                break;
            case 3:
                total = quantidade * 5;
                String t3 = String.format("%.2f", total);
                System.out.println("Total: R$ : " + t3);
                break;
            case 4:
                total = quantidade * 2;
                String t4 = String.format("%.2f", total);
                System.out.println("Total : R$ " + t4);
                break;
            case 5:
                total = quantidade * 1.5;
                String t5 = String.format("%.2f", total);
                System.out.println("Total : R$ " + t5);
                break;

            default:
                break;
        }

    }
}
